controller/system_backup_controller.go

```diff
@@ -422,10 +426,7 @@ func (c *SystemBackupController) reconcile(name string, backupTargetClient engin
 		cleanupLocalSystemBackupFiles(tempBackupArchivePath, tempBackupDir, log)
 
 	case longhorn.SystemBackupStateDeleting:
-		err = c.cleanupRemoteSystemBackupFiles(systemBackup, backupTargetClient)
-		if err != nil {
-			return err
-		}
+		cleanupRemoteSystemBackupFiles(systemBackup, backupTargetClient, log)
```

Comment:
So we will ignore the remote deletion if there is a connection issue with the backup target. It means the SystemBackup CR will be deleted, but later if the connection is backup, the SystemBackup will be resync from the target, and be created with ready state, right?

Reply:
Correct.